app/controllers/api/verify/password_confirm_controller.rb

```diff
@@ -42,15 +42,22 @@ def add_proofing_component(user)
         ProofingComponent.create_or_find_by(user: user).update(verified_at: Time.zone.now)
       end
 
-      def completion_url(result)
-        if result.extra[:profile_pending]
+      def completion_url(result, user)
+        if in_person_enrollment?(user)
+          idv_in_person_ready_to_verify_url
+        elsif result.extra[:profile_pending]
           idv_come_back_later_url
         elsif current_sp
           sign_up_completed_url
         else
           account_url
         end
       end
+
+      def in_person_enrollment?(user)
+        # WILLFIX: After LG-6708 and we have enrollment saved, reference enrollment instead.
+        ProofingComponent.find_by(user: user)&.document_check == Idp::Constants::Vendors::USPS
+      end
```

Comment:
I'm going to add some configuration-based bailouts here, to avoid a database query in production when we're not expecting there to be any in-person enrollments.

Reply:
Updated in f4da2ff8202876cb6e38d33f51d179b96d397ad5.